example/lib/view/home.dart

```diff
@@ -0,0 +1,92 @@
+import '../utils/dimension_constants.dart';
+import '../utils/string_constants.dart';
+import '../route/route_constants.dart';
+import 'package:flutter/material.dart';
+
+class HomePage extends StatefulWidget {
+  const HomePage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          StringConstants.mainAppBarTitle,
+        ),
+        centerTitle: true,
+      ),
+      body: GridView.count(
+        crossAxisCount: DimensionConstants.gridViewCrossAxisCount,
+        children: [
+          Column(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              ElevatedButton(
+                onPressed: () {
+                  Navigator.pushNamed(
+                    context,
+                    RouteConstants.widgetTitle,
+                  );
+                },
+                child: Text(
+                  StringConstants.titleWidget,
+                ),
+              ),
```

Comment:
This is only the code that you are going to have in the return of ListBuilder

Reply:
Implemented, thank you.